### PR TITLE
Xpouyat july2021

### PR DESCRIPTION
**Pull request by xpouyat**
## Purpose

* Code moved to .Net 5
* Support for .env file with documentation
* Support for VS Code
* Nugget package update

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: AMSV3Quickstarts/EncodeAndStreamFiles/Program.cs [hunk 1]
```diff
@@ -35,7 +47,7 @@ public static async Task Main(string[] args)
             {
                 if (exception.Source.Contains("ActiveDirectory"))
                 {
-                    Console.Error.WriteLine("TIP: Make sure that you have filled out the appsettings.json file before running this sample.");
+                    Console.Error.WriteLine("TIP: Make sure that you have filled out the .env or appsettings.json file before running this sample.");
```
Comment (johndeu) [inline AMSV3Quickstarts/EncodeAndStreamFiles/Program.cs:50]: Maybe say, copy the sample.env and rename to .env prior to filling it out. We could also warn not to enter secrets into the sample.env file - as that could get checked back into a fork of the github repo.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ To run samples in this repository, you need:
 
 * Visual Studio 2019 or Visual Studio Code
 * An Azure Media Services account. See the steps described in [Create a Media Services account](https://docs.microsoft.com/azure/media-services/latest/create-account-cli-quickstart).
+* Create and fill a .env file based on the [sample file](AMSV3Quickstarts/sample.env), or fill the [appsettings.json](AMSV3Quickstarts/EncodeAndStreamFiles/appsettings.json) file. Be sure to copy the API Access details which are in the Azure portal for the AMS account.
```
Comment (johndeu) [inline README.md:33]: Add more warnings here.

We want folks to copy the sample.env to a .env file first... and make sure that they never enter secrets into the sample.env file directly and check that one in.  We can state that the .env file is safe, as it is ignored using the .gitignore file

File: AMSV3Quickstarts/EncodeAndStreamFiles/Program.cs (hunk 1 above)
Comment (xpouyat) [reply AMSV3Quickstarts/EncodeAndStreamFiles/Program.cs:50]: It's noted in sample.env file https://github.com/Azure-Samples/media-services-v3-dotnet-quickstarts/commit/b5c20f11116f91c014d0b66ba54a1302449f92e1

File: README.md (hunk 2 above)
Comment (xpouyat) [reply README.md:33]: Again, it's noted in sample.env file https://github.com/Azure-Samples/media-services-v3-dotnet-quickstarts/commit/b5c20f11116f91c014d0b66ba54a1302449f92e1

Comment (xpouyat) [reply README.md]: https://github.com/Azure-Samples/media-services-v3-dotnet-quickstarts/commit/15ae4f019d9939bfdf3d06253f25f83e0c601a40

**Pull request merged**